### PR TITLE
feat: resend provider

### PR DESCRIPTION
**Pull request by talboren**

### CODE REVIEW

File: keep/providers/resend_provider/resend_provider.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from keep.providers.base.base_provider import BaseProvider
+from keep.providers.models.provider_config import ProviderConfig
+
+
+@pydantic.dataclasses.dataclass
+class ResendProviderAuthConfig:
+    api_key: str = dataclasses.field(
+        metadata={
+            "required": True,
+            "description": "Resend API key (https://resend.com/api-keys)",
```
Comment (shahargl) [inline keep/providers/resend_provider/resend_provider.py:18]:
```
"description": "Resend API key",
"hint": "https://resend.com/api-keys"
```

File: keep/providers/resend_provider/resend_provider.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    # Initalize the provider and provider config
+    config = ProviderConfig(
+        id="resend-test",
+        authentication={"api_key": resend_api_key},
+    )
+    provider = ResendProvider(provider_id="resend-test", config=config)
+    response = provider.notify(
+        "onboarding@resend.dev",
+        "youremail@gmail.com",
+        "Hello World!",
```
Comment (shahargl) [inline keep/providers/resend_provider/resend_provider.py:94]: Something with Keep? "Hello World from Keep!"

**Review by shahargl — APPROVED** [review]
lgtm, had two comments

File: keep/providers/resend_provider/resend_provider.py (hunk 2 above)
Comment (talboren) [inline keep/providers/resend_provider/resend_provider.py:94]:
```suggestion
        "Hello World from Keep!",
```

Comment (talboren) [reply keep/providers/resend_provider/resend_provider.py]: Fixed

File: keep/providers/resend_provider/resend_provider.py (hunk 1 above)
Comment (talboren) [reply keep/providers/resend_provider/resend_provider.py]: Fixed :)

**Pull request merged**